tests/ChunkDecoder/PressureChunkDecoderTest.php

```diff
@@ -21,12 +21,14 @@ protected function setup()
      * @param string $remaining
      * @dataProvider getChunk
      */
-    public function testParse($chunk, $pressure, $pressure_unit, $remaining)
+    public function testParse($chunk, $pressure, $pressure_hPa, $pressure_inHg, $pressure_unit, $remaining)
```

Comment:
It's very good that you write tests. However, I think it would be simpler to write all these test cases on the Value class alone. The main advantage is that you can test any kind of values and conversion easily if you're testing the Value class alone.

Test on chunk decoders are here to check the parsing, the conversion is totally independant. I propose to keep tests on conversion on the Value class in order to keep chunk tests simple. One test on the overall decoding could be interesting though.